.eslintrc

```diff
@@ -1,176 +1,176 @@
 {
```

Comment:
1. Firstly, it is not actually needed. ESLint by itself supports ES6 linting pretty well. Babel transformer slightly changes the code and some rules fail after those changes (I experienced it with my stricter linting config)
2. In the project there is 2 spaces indentation everywhere, so it is a little bit odd to see 4 spaces here (need .editorconfig #4)